README.md

```diff
@@ -24,6 +24,7 @@ The repository is based on the [template-typescript-package](https://github.com/
 **CD** 🚀
 
 ![Release](https://github.com/UK-Export-Finance/exip/actions/workflows/deployment.yml/badge.svg?branch=dev)
+![Release](https://github.com/UK-Export-Finance/exip/actions/workflows/deployment.yml/badge.svg?branch=feature)
```

Comment:
Should we add some context to all release badges? Otherwise it's just some nice looking badges, might seem they're accidentally duplicated? e.g

<img width="472" alt="Screenshot 2024-02-15 at 17 02 48" src="https://github.com/UK-Export-Finance/exip/assets/2463648/aac6f4f2-6d5d-4236-895e-f62165034849">


@abhi-markan  ? Maybe similar in other repos.

Reply:
there does not seem to be a way to change the text unless we use html for these badges

Reply:
I was thinking could just have a list of a heading for each environment

Reply:
Yes unfortunately we cannot change the text however we add them under bullet points.

Reply:
I'll try some things out if that's OK @abhi-markan , can try having a list format with each environment listed.

Would also be good to have this in the `main` branch and not just `no-pdf`

Reply:
How about this @abhi-markan ? Table format with readable context for each environment.

Could be a standard for all repos?

<img width="640" alt="Screenshot 2024-02-19 at 17 11 16" src="https://github.com/UK-Export-Finance/exip/assets/2463648/96549542-abcb-471b-8beb-062a15023a02">

Reply:
Looks good :-)